.github/workflows/integration.yml

```diff
@@ -89,22 +89,6 @@ jobs:
               go env -w GOFLAGS=-mod=mod
               make ci-e2e
 
-      - name: install tools
-        run: make install
-
-      - name: Build docker image
-        run: |
-          KO_PREFIX=gcr.io/openssf/scorecard make dockerbuild
-
-      - name: Test docker image
-        env:
-          GITHUB_AUTH_TOKEN: ${{ secrets.GH_AUTH_TOKEN }}
-        run: |
-              echo "Skipping e2e tests. Will be re-enabled soon."
-              #mkdir $OUTPUT_PATH
-              #docker run -e GITHUB_AUTH_TOKEN=$GITHUB_AUTH_TOKEN  $IMAGE_NAME  --show-details  --repo=https://github.com/ossf/scorecard --metadata=openssf --format json > ./$OUTPUT_PATH/results.json
-              #ginkgo -p -v --focus="E2E TEST:executable" ./e2e/...
-
```

Comment:
Why are we removing this? Is it because `make build` will run during the `Build` step of the GitHub workflow?

Reply:
It is already there in the other action. So this is a duplicate.

Reply:
https://github.com/ossf/scorecard/blob/6562cc1f4488c7a018b5c6b4e031f990058d95a2/.github/workflows/main.yml#L41 

The `make all` has the build step.